### PR TITLE
Pre-reqs and steps updated to better reflect current process

### PR DESCRIPTION
**Pull request by jms-pantheon**
## Summary

**[Pantheon Front-End Sites - Site Creation](https://pantheon.io/docs/guides/decoupled-sites/)** - The steps occasionally had confusing information about repositories, and the tabs had overlapping IDs. I tried to run through each scenario and make sure it matches up and clarify as needed

**Release**:
- [x] When ready
- [ ] After date: $DATE

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

File: source/content/guides/decoupled-sites/04-site.md [hunk 1]
```diff
@@ -16,13 +16,15 @@ reviewed: "2022-09-19"
 
 ## Before You Begin
 
-You should have the **Decoupled** tab enabled in your dashboard after onboarding. To begin creating a Front-End Site, ensure you meet the following prerequisites:
+You should have the **Decoupled** tab enabled in your Sites dashboard after onboarding. To begin creating a Front-End Site, ensure you meet the following prerequisites:
 
-* Your site repository is in GitHub.
+* You have a GitHub account
```
Comment (logickal) [inline source/content/guides/decoupled-sites/04-site.md:21]: Probably worth mentioning as well that if they are bringing an existing codebase that the site repository exists on GitHub.

File: source/content/guides/decoupled-sites/04-site.md [hunk 2]
```diff
@@ -109,18 +111,20 @@ Create a site without configuring the backend and link to a frontend framework.
...
 
     * Specify the Site Name: The site name is the title of your site. You can edit the site name in **Settings** after creation. 
-    * Select the frontend framework. You can choose either Next.js or Gatsby as the static site generator.
+    * Select the frontend framework. You can choose either Next.js or Gatsby as the static site generator. 
+        If you are testing out a static React.js application, you may try selecting Gatsby. If the application is using the Node runtime then you will want to select Next.js as the framework. Please note we will not be able to provide support for frameworks outside of Next.js or Gatsby.
 
 1. (Optional) Link your CMS.
 
-    > **NOTE**: If you are using a specified CMS, you can link your CMS backend and the site environment from which to source content. However, this is more applicable when using the Decoupled starter kit templates.
+    > **NOTE**: If you are using a back-end CMS on Pantheon, you can link your CMS and the site environment from which to source content. However, this is optional and more applicable when using the Decoupled starter kit templates.
```
Comment (logickal) [inline source/content/guides/decoupled-sites/04-site.md:127]: I have some quibbles with this section that I have been struggling with.  From an EA standpoint, we definitely have customers who are potentially bringing sites over that are already configured to address a CMS.  I think we need to expand on documentation for that use case.

File: source/content/guides/decoupled-sites/04-site.md (hunk 1 above)
Comment (jms-pantheon) [reply source/content/guides/decoupled-sites/04-site.md:21]: To me that's a specific qualification for doing the Import workflow, and that section has its own pre-reqs. I think that's a bit of a self-fulfilling one, as if they are importing a repo they must have a repo. Happy to change it if you see it differently of course

File: source/content/guides/decoupled-sites/04-site.md (hunk 2 above)
Comment (jms-pantheon) [reply source/content/guides/decoupled-sites/04-site.md]: That makes sense, I've just expanded this section in [dad37e8](https://github.com/pantheon-systems/documentation/pull/7715/commits/dad37e832fdbb14cd74760037b70ddc804df63d7)

**Comment by joa-pan** [comment]
lgtm

**Pull request merged**